Minimum_inversions/Minimum_inversions.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+    Minimum_Inversions
+   Given a binary array array[] of size num. The task is to find the minimum number of
+   inversions required so that no two adjacent elements are same.
+*/
+#include <bits/stdc++.h>
+using namespace std;
+
+// Function to return the minimum
+// inversions required so that no
+// two adjacent elements are same
+int min_changes(int a[], int n)
+{
+	int ans_a = 0, ans_b = 0;
+
+	// Find all the changes required
+	for (int i = 0; i < n; i++) {
+		if (i % 2 == 0) {
+			if (a[i] == 0)
+				ans_a++;
+			else
+				ans_b++;
+		}
+		else {
+			if (a[i] == 0)
+				ans_b++;
+			else
+				ans_a++;
+		}
+	}
+
+
+	// Return the required answer
+	return min(ans_a, ans_b);
+}
+
+int main()
+{
+    int num;
+    cout<<"Enter number of elements in your array: ";
+    cin>>num;
+    int array[num];
+    cout<<"Enter your array: ";
+    for (int i = 0; i < num; i++)
+    {
+        cin>>array[i];
+    }
+
+	cout <<"Minimum changes required : "<< min_changes(array, num);
+
+	return 0;
+}
```

Comment:
keep a line space before the commend

Reply:
@asha15  Required changes done please review it.